src/main/java/org/craftercms/studio/api/v2/content/LifeCycleContent.java

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright (C) 2007-2025 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.craftercms.studio.api.v2.content;
+
+import org.apache.commons.io.FilenameUtils;
+import org.craftercms.studio.api.v2.repository.ContentWriteItem;
+import org.dom4j.Document;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Collections.unmodifiableMap;
+import static org.craftercms.studio.api.v2.content.LifeCycleContentProvider.ofPath;
+import static org.craftercms.studio.api.v2.utils.StudioUtils.createTempFile;
+
+/**
+ * Container for content items to be passed to the content life cycle controller
+ */
+public class LifeCycleContent implements AutoCloseable {
```

Comment:
Lifecycle is a single word. We should not camelCase it nor separate it.

Reply:
Updated